app/src/main/java/com/example/dazn_movie_player/models/services/MovieApiInterface.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.dazn_movie_player.models.services
+
+import com.example.dazn_movie_player.models.MovieResponse
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface MovieApiInterface {
+
+    @GET("/3/movie/popular?api_key=ae3e51a2cc95161ed1c94b4e989ed9f7")
```

Comment:
I would suggest using a constant variable
```suggestion
    @GET(someRelevantVariable)
```